### DISCUSSION TITLE
Slashcommand enhancement

### DISCUSSION THREAD

**Discussion by CGraabaek** [arnoudkooi/SN-Utils#510] [Q&A] [answered]
Is it possible to add a slashcommand for going to the currently selected updateset?

**Comment by arnoudkooi** [comment, marked as answer]
You can add a custom one, see the /cus command in this set of examples
https://github.com/TWalkerVeracity/SlashcommandArchive
(The SN Devs slack channel #snutils is also a great place to discuss questions like this, just a tip, feel free)

**Reply by CGraabaek** [reply]
Thanks! That's a really great collection, I knew about custom slashcommands but just thought it would be something everybody could benefit from.
